packages/manager/apps/web-office/src/api/license/api.ts

```diff
@@ -1,12 +1,12 @@
 import { v6 } from '@ovh-ux/manager-core-api';
-import { GetOfficeLicenseServiceParams } from './type';
 import { getApiPath, getApiPathWithoutServiceName } from '../utils/apiPath';
+import { LicenseType } from './type';
 
 // GET
 
-export const getlicenseOfficeServiceQueryKey = (
-  params: GetOfficeLicenseServiceParams,
-) => [`get/license/office/${params.serviceName}`];
+export const getlicenseOfficeServiceQueryKey = (serviceName: string) => [
+  `get/license/office/${serviceName}`,
```

Comment:
**nitpick**: prefer declaring key like that `['get', 'license', 'office', serviceName]`